index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>  
+    
+    <section class="firstPart">
+        <header class="header">
+        <nav class="navbar">
+        <div class="container">
+            <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+</nav>
```

Comment:
The navbar code is not properly indented, making it difficult to read and maintain. Properly indenting the code will improve its readability and make it easier to modify in the future.

Reply:
- [ ] You have done a great job💪 explaining why you suggest changes♻. However, when suggesting changes, it is recommended to use more polite😘 language. Instead of saying `The navbar code is not properly indented,` you could consider starting with `🙏Please consider making changes to the navbar code`.This will make the person🤩 whom you give a review feel respected for what they have done and encourage them to implement the changes.